polyA/printers.py

```diff
@@ -179,14 +180,19 @@ def print_results_soda(
     consensus_matrix_collapse: Dict[Tuple[int, int], int],
     subfams_collapse_index: Dict[str, int],
     node_confidence_orig: Dict[Tuple[str, int], float],
-    ids: List[str],
+    ids: List[int],
+    subfam_alignments: List[str],
+    chrom_alignments: List[str],
+    subfam_alignments_collapse: Dict[Tuple[str, int], Tuple[int, int]],
```

Comment:
Can we create a type alias for this and use it instead of spelling out the whole thing in various places? See the alias for `SupportMatrix` for an example.